src/cheri/introduction.adoc

```diff
@@ -37,6 +37,7 @@ extensions to support CHERI alongside RISC-V
 
 <<rv32y,{cheri_base_ext_name}>>:: Unprivileged specification for {cheri_base_ext_name} to support CHERI.
 <<section_rvy_sentry_insn_ext>>:: Unprivileged extension for capability ambient sealing.
+<<section_rvy_sentry_jalr_ext>>:: Unprivileged extension for sealed function entry points (for CFI).
```

Comment:
Can we have this extension be a hook in the JALR/xRET instructions instead of a separate named extension?